clusters/malpedia.json

```diff
@@ -13,7 +13,7 @@
   "name": "Malpedia",
   "source": "Malpedia",
   "type": "malpedia",
-  "uuid": "1d1c9af9-37fa-4deb-a928-f9b0abc7354a",
```

Comment:
Not sure why you changed the UUID, it should remain fixed.

Reply:
hm Curious .. this came automaticly from the [fetch_malpedia.sh](https://github.com/MISP/misp-galaxy/blob/main/tools/fetch_malpedia.sh) script

Reply:
@danielplohmann maybe we should keep the UUID of the cluster from the malpedia api. What do you think?

Reply:
I just had a look at our code base, the generated galaxy accessible from our API endpoint should always come with UUID `5fc98d08-90a4-498a-ad2e-0edf50ef374e` for consistency reasons, only the version changes in accordance with our internal commit count.

Reply:
Ok, so that means @danielplohmann  that `5fc98d08-90a4-498a-ad2e-0edf50ef374e` are the correct UUID ?

Reply:
@danielplohmann It should be the current one in the misp repository. If not it will be see as a new malpedia galaxy. Could you use the current one? Thank you

Reply:
Yeah, no problem, we will change it on our server to match the UUID that you already have!
Should be deployed in the next days with an upcoming update we are preparing anyway. :+1:

Reply:
for completeness, we finally deployed a big update yesterday which should also fix the UUID issue that was raised here.